api/src/pcapi/routes/backoffice/users/blueprint.py

```diff
@@ -230,3 +235,36 @@ def confirm_batch_suspend_users() -> utils.BackofficeResponse:
         flash(f"{len(users)} compte d'utilisateur a été suspendu", "success")
 
     return redirect(url_for("backoffice_web.fraud.list_blacklisted_domain_names"), code=303)
+
+
+@users_blueprint.route("/<int:user_id>/redirect_to_brevo", methods=["GET"])
+@utils.permission_required_in(
+    [
+        perm_models.Permissions.READ_PUBLIC_ACCOUNT,
+        perm_models.Permissions.READ_PRO_ENTITY,
+    ]
+)
+def redirect_to_brevo_user_page(user_id: int) -> utils.BackofficeResponse:
+    user = (
+        users_models.User.query.filter_by(id=user_id).options(joinedload(users_models.User.UserOfferers)).one_or_none()
+    )
+
+    if not user:
+        raise NotFound()
+    try:
+        contact_info = mails.get_contact_info(user.email)
+    except ExternalAPIException as exp:
+        if exp.is_retryable:
+            flash("L'utilisateur ne semble pas être dans Brevo", "warning")
```

Comment:
Soyons sûrs de la requête de recherche !
```suggestion
            flash(Markup("L'utilisateur <b>{email}</b> n'existe pas Brevo").format(email=user.email), "warning")
```